plugins/kiali/DEVELOPMENT.md

```diff
@@ -112,14 +100,22 @@
    Add to locations in `app-config.local.yaml`
 
    ```yaml
+   backend:
+     auth:
+       dangerouslyDisableDefaultAuthPolicy: true
+   ---
    locations:
```

Comment:
This text goes below in the Add catalog section.

Reply:
Thanks! Updated